subset/bacnet/bacnetTests/src/main/java/VersionTest.java

```diff
@@ -131,10 +133,10 @@ private void formatProperty(Map.Entry<PropertyIdentifier, Encodable> property) {
   private void generateReport(String deviceMacAddress) {
```

Comment:
@JoelRedstone @ewRedstone @ktredstone 

Just a style point - I've noticed that @JoelRedstone, in this test, has a set of classes (e.g. Report.java) that deal with things that you're doing in every single test; things like writing reports, handling connections and checking input parameters etc.

To make your lives easier, you guys could potentially have a `helper` set of functions in the `subset/` folder itself to standardise all of these things that are to do with DAQ specfic stuff so you can focus on functional things?

Reply:
Very good idea!